src/shadowbox/server/api.yml

```diff
@@ -27,7 +27,36 @@ paths:
               examples:
                 '0':
                   value: >-
-                    {"name":"My Server","serverId":"000-000-000-000","metricsEnabled":true,"createdTimestampMs":1536613192052,"portForNewAccessKeys":1234}
+                    {"name":"My Server","serverId":"000-000-000-000","metricsEnabled":true,"createdTimestampMs":1536613192052,"portForNewAccessKeys":1234, "version": "1.0.0"}
+  /server/hostname-for-new-access-keys:
+    put:
+      tags:
+        - Server
+      description: Changes the hostname for new access keys.  Must be a valid hostname or IP address.  If it's a hostname, DNS must be set up independently of this API.
+      requestBody:
+        required: true 
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                hostname:
+                  type: string
+            examples:
+              'hostname':
+                value: '{"hostname": www.example.org}'
+              'IP address':
+                value: '{"hostname": "127.0.0.1"}'
+      responses:
+        '204': 
+          description: The hostname was successfully changed.
+        '400':
+          description: An invalid hostname or IP address was provided.
+        '404':
+          description: The given hostname doesn't point to an equivalent Shadowbox server
```

Comment:
From the offline discussion, I think we'll move this to the manager, so remove the error?

Reply:
Done